From ac19e6fd755bd143f41fd84ce9378c1e55953e64 Mon Sep 17 00:00:00 2001
From: Robert Tiemann <r.tiemann@ta-hifi.com>
Date: Thu, 20 Feb 2020 11:39:37 +0100
Subject: [PATCH] Makefile: Remove leading backslash from #include.

Seems like most recent glib-mkenums doesn't like it anymore. Or GNU
Make. Or whatever program is processing this madness.
---
 common/gst-glib-gen.mak | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/gst-glib-gen.mak b/common/gst-glib-gen.mak
index 0b0a5e9..93bb28c 100644
--- a/common/gst-glib-gen.mak
+++ b/common/gst-glib-gen.mak
@@ -6,7 +6,7 @@
 #glib_gen_prefix=gst_color_balance
 #glib_gen_basename=colorbalance
 
-enum_headers=$(foreach h,$(glib_enum_headers),\n\#include \"$(h)\")
+enum_headers=$(foreach h,$(glib_enum_headers),\n#include \"$(h)\")
 
 # these are all the rules generating the relevant files
 $(glib_gen_basename)-marshal.h: $(glib_gen_basename)-marshal.list
-- 
2.25.1

